services/friendbot/internal/friendbot_handler.go

```diff
@@ -94,8 +96,8 @@ func (handler *FriendbotHandler) loadAmount(r *http.Request) (string, error) {
 	return unescaped, err
 }
 
-func (handler *FriendbotHandler) loadResult(address string, amount string) (*horizon.TransactionSuccess, error) {
-	result, err := handler.Friendbot.Pay(address, amount)
+func (handler *FriendbotHandler) loadResult(address string, amount string, isFundRequest bool) (*horizon.TransactionSuccess, error) {
+	result, err := handler.Friendbot.Pay(address, amount, isFundRequest)
```

Comment:
why is pay called inside `loadResult` and not in the handler function?

Reply:
Because the Stellar devs wrote it that way.